### PR TITLE
Update translation

### PR DESCRIPTION
**Pull request by dhermes**

### CODE REVIEW

File: translate/google/cloud/translate/client.py [hunk 1]
```diff
@@ -173,7 +185,11 @@ def translate(self, values, target_language=None, format_=None,
                                   for translation. Sets the ``cid`` parameter
                                   in the query.
 
-        :rtype: str or list list
+        :type model: str
+        :param model: (Optional) The model used to translate the text. The
+                      only accepted values are ``base`` and ``nmt``.
```
Comment (jonparrott) [inline translate/google/cloud/translate/client.py]: Should we have constants for these?

File: translate/google/cloud/translate/connection.py [hunk 2]
```diff
@@ -20,11 +20,14 @@
...
+    API_BASE_URL = 'https://translation.googleapis.com'
     """The base of the API call URL."""
 
     API_VERSION = 'v2'
     """The version of the API, used in building the API call's URL."""
 
     API_URL_TEMPLATE = '{api_base_url}/language/translate/{api_version}{path}'
     """A template for the URL of a particular API call."""
+
+    SCOPE = ('https://www.googleapis.com/auth/cloud-platform',)
```
Comment (dhermes) [inline translate/google/cloud/translate/connection.py]: I don't think the service has its own scope yet.

Comment (jonparrott) [inline translate/google/cloud/translate/connection.py]: Ew. Alright.

File: translate/google/cloud/translate/client.py (hunk 1 above)
Comment (jonparrott) [inline translate/google/cloud/translate/client.py]: Please do, I definitely prefer that over magic strings. Unless there is some way to get the list of models from the API?

**Comment by dhermes** [comment]
@jonparrott PTAL

**Comment by dhermes** [comment]
We'll sort out the API key issue after the release (joy).

**Pull request merged**